docs/reference/ig.md

```diff
@@ -27,15 +27,13 @@ Some characteristics of `ig`:
 
 ## Use cases
 
-- In a Kubernetes environment, when the Kubernetes API server is not working
-  properly, we cannot deploy Inspektor Gadget. Therefore, we still need a way to
-  debug the containers of the cluster.
-- In some cases, you might have root SSH access to the Kubernetes nodes of a
-  cluster, but not to the `kubeconfig`.
+- In a Kubernetes environment, even when the Kubernetes API server is working properly, you might prefer to trace
+  containers without relying on Kubernetes itself.
+- In some cases, you might have root SSH access to the Kubernetes nodes of a cluster, but not to the `kubeconfig`.
 - If you don't want to install `kubectl-gadget` on your machine, you can run
-  `ig` in a Kubernetes pod and read the output directly.
-- If you are implementing an application that needs to get insights from the
-  Kubernetes node, you could include the `ig` binary in your container
+  `ig` in a Kubernetes pod and read the   output directly.
+- If you are implementing an application that needs to get insights from the Kubernetes node, you could include the `ig` binary 
+  in your container
   image, and your app simply execs it. In such a case, it is suggested to use
   the JSON output format to ease the parsing.
 - Outside a Kubernetes environment, for observing and debugging standalone
```

Comment:
I'm not sure about these changes. Please drop them, I'll try to work later on on that part.